scanner/scanner.go

```diff
@@ -306,8 +306,14 @@ func (s *Scanner) scanDoubleQuote(ctx *Context) (tk *token.Token, pos int) {
 		c := src[idx]
 		pos = idx + 1
 		ctx.addOriginBuf(c)
+		var nextChar rune
+		if idx+1 < size {
+			nextChar = src[idx+1]
+		}
 		if s.isNewLineChar(c) {
-			value = append(value, ' ')
+			if nextChar != '"' {
```

Comment:
Please explain why this branching is necessary.

Reply:
Fair point, I don't remember so I'll play with the tests to see if I can reproduce the original issue and provide context.

Reply:
This is an error on my part, the code was correct.